locales/tr-TR/translation.json

```diff
@@ -0,0 +1,280 @@
+  "I have read and understood the documentation": "Dökümanı okudum ve anladım",
+  "Use optional passphrase (advanced)": "Opsiyonel kurtarma sözcükleri kullan (gelişmiş)",
+  "Confirm passphrase": "Kurtarma sözcüklerini doğrula",
+  "Passphrases don't match": "Kurtarma sözcükleri uyuşmuyor",
+  "Language": "Dil",
+  "Change the wallet language": "Cüzdan dilini değiştirin",
+  "Activate dark mode": "Karanlık modu aktifleştir",
+  "Pick a color": "Renk seç",
+  "Close": "Kapat",
+  "Tab navigation": "Navigasyon sekmesi",
+  "Make this your default address": "Bunu varsayılan adresiniz yapın",
+  "into": "buraya",
+  "out from": "buradan",
+  "Transaction details": "İşlem detayları",
+  "Copy address": "Adresi kopyala",
+  "Custom": "Uyarlama",
+  "Moved": "Taşındı",
+  "Unlocks at": "Kilidi açılır",
+  "Unlocked at": "Kilidi açıldı",
+  "Label": "Etiket",
+  "Last used": "Son kullanım",
+  "Transactions": "İşlemler",
+  "ALPH amount": "ALPH miktarı",
+  "Set lock time": "Kilit süresini ayarla",
+  "Tweak gas settings": "Gaz seçeneklerini ayarla",
+  "Call contract": "Sözleşmeyi çağır",
+  "Deploy contract": "Sözleşmeyi teslim et",
+  "Issue token amount": "Belirlenen varlık miktarı",
+  "Tokens to issue (optional)": "Yayınlanacak varlıklar (tercihe bağlı)",
+  "Gas": "Gaz",
+  "Review": "Gözden geçir",
+  "Invalid address group for the WallectConnect connection": "WalletConnect bağlantısı için hatalı adres grubu",
+  "Error in pairing": "Eşleştirme hatası",
+  "WalletConnect error": "WalletConnect hatası",
+  "Try again": "Tekrar dene",
+  "Connect to a dApp": "dApp (dijital uygulama) bağla",
+  "Paste what was copied from the dApp": "dApp (dijital uygulama) tarafından kopyalananları yapıştır",
+  "Connect": "Bağlan",
+  "Approve the proposal to connect": "Bağlanma talebini onaylayın",
+  "Please review the following before authorizing the dApp": "dApp (dijital uygulama) yetkilendirmesini tamamlamadan önce lütfen aşağıdakileri gözden geçirin",
+  "Name": "İsim",
+  "URL": "URL (bağlantı kaynağı",
+  "Description": "Tanım",
+  "Network ID": "Network ID (Ağ kimliği)",
+  "Address group": "Adres grubu",
+  "all": "tümü",
+  "Signer address": "İmzalayıcı adres",
+  "Select an address to sign with.": "İmzalayıcı adresi seçin.",
+  "Reject": "Reddet",
+  "Approve": "Onayla",
+  "Unable to convert the amount": "Miktar dönüştürülemiyor",
+  "The amount must be greater than {{ minAmount }}": "Miktar {{ minAmount }} adetten daha fazla olmalıdır.",
+  "Restart the application to install the new update.": "Yeni güncellemeyi yüklemek için uygulamayı yeniden başlatın.",
+  "Downloaded {{ percent }}%": "%{{ percent }} indirildi",
+  "Update": "Güncelle",
+  "Download failed, try again later.": "İndirme başarısız, daha sonra tekrar deneyin.",
+  "Optional passphrase": "Tercihe bağlı kurtarma sözcükleri",
+  "Pending": "Bekliyor",
+  "Mainnet": "Mainnet (Ana ağ)",
+  "Testnet": "Testnet (Test ağı)",
```

Comment:
Hm, not a big fan of having both English and Turkish strings in the translation... Let's choose one of the two.


<img width="532" alt="image" src="https://user-images.githubusercontent.com/1579899/218670149-17ee74b1-747b-4e78-9828-042d6c87498f.png">

Reply:
@nop33 hi. I am understanding on your confusion why require use original and explain parenthetical. All of us know that common usage of techonological (computer terms) specific calls. Also we are in age of crypto more extented tech. For explaining some of terms, we are not having mean directly whats it in Turkish dictionary tbh. Because of even after crypto joins to era, there ere more specific names appeared. Example on Url defines shorter of Uniform Resource Locator but if i put shorter in Turkish due that's non common usage in Turkish no one understands that. We still use it like url as url defining like i mentioning web address in Turkish bağlantı kaynağı or web adresi ismi. So others like dApp Expented digital application prefers dijital uygulama but i can't call it like duyg cause no one will understand what it really defines and some meaningless. Also not common used short term. Other examples api network id mainnet testnet genesis tx. We can use original ones if you wish to keep areas clean because of like i said they are special terms for crypto tech so refers common understanding. I just tried to define all of them in Turkish. Of course As i am not expert linguist or language trainer but i think Turkish people may understanding what those original terms works and explains.

Reply:
> Example on Url defines shorter of Uniform Resource Locator but if i put shorter in Turkish due that's non common usage in Turkish no one understands that. 

Of course, a Turkish abbreviation of "URL" would indeed be confusing. How about simply "URL"? Isn't that a well-established technological term? I believe it is!

> So others like dApp Expented digital application prefers dijital uygulama

dApp does not stand for "digital application" but for "decentralized application". My opinion is that it is not the job of the wallet interface to educate the user regarding what "dApp" means. I find it irrelevant in the context of our UI. Besides, if someone does not know its meaning, it's preferable to not use the feature, and head to some doc page to get a better understanding before using it. Of course, I believe that we should make an effort to educate our users through [our docs](https://docs.alephium.org/), but I also believe in a clean and easy-to-digest UI. Adding explanations of terms in parenthesis in the UI clatters it imo.

I think you've done a great job, and it's admirable that you are caring a lot for explaining to the user terms they might not understand! My opinion is that although these explanations are useful, our UI it's not the right place for them.

Reply:
I agree on your vision. And i checked your ss examples it keeps mass detail and longer. So any can easy look in for more detailed documentation info. Well so i' m diving back inside of translation and removing explained details to keep this clean. 

I' m really happy to help grow in Alephium project, cause one of my best believing project to have it in good future. Really thanks for your good thoughts.